internal/mesh/internal/types/mesh_configuration.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import (
+	"github.com/hashicorp/consul/internal/resource"
+	pbmesh "github.com/hashicorp/consul/proto-public/pbmesh/v2beta1"
+)
+
+// RegisterMeshConfiguration takes
```

Comment:
think this is missing the rest of this comment

Reply:
RegisterMeshConfiguration takes and it takes and it takes.